pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocGjenlevenderettFoed1954Til1957.kt

```diff
@@ -0,0 +1,171 @@
+                            + " The supplement, therefore, will be gradually reduced until it ceases to exist. When the supplement has been removed,"
+                            + " your retirement pension will increase as normal with the annual adjustment again."
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Hvis du tar ut mindre enn 100 prosent alderspensjon, har det betydning for gjenlevendetillegget ditt."
+                            + " Ved senere økning av uttaksgrad vil ikke gjenlevendetillegget økes.",
+                    Nynorsk to "Viss du tar ut mindre enn 100 prosent alderspensjon, verkar det inn på attlevandetillegget ditt."
+                            + " Viss du aukar uttaksgraden seinare, vil ikkje attlevandetillegget auke.",
+                    English to "If you take out less than 100 percent retirement pension, this will affect your survivor's supplement."
+                            + " If you should later on increase the withdrawl rate of your retirement pension, the survivor's supplement wil not increase."
+                )
+            }
+
+            title1 {
+                text(
+                    Bokmal to "Hva vil skje i saken din?",
+                    Nynorsk to "Kva vil skje i saka di?",
+                    English to "What will happen in your case?"
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Mot slutten av dette året får du et brev som viser hva som er gjenlevendetillegg etter gamle og nye regler og hva som er alderspensjonen din.",
+                    Nynorsk to "Mot slutten av dette året får du eit brev som viser kva som er gjenlevendetillegg etter gamle og nye reglar og kva som er alderspensjonen din.",
+                    English to "Towards the end of the year, you will receive a letter that informs of what is your survivor's rights and what is your retirement pension."
+                )
+            }
+
+            title1 {
+                text(
+                    Bokmal to "Du må melde fra om endringer",
+                    Nynorsk to "Du må melde frå om endringar",
+                    English to "Notify NAV about changes"
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Du mister gjenlevenderetten hvis du gifter deg, flytter sammen med en du tidligere har vært gift med eller har barn med, eller får barn med en samboer.",
+                    Nynorsk to "Du mistar attlevandetillegget om du giftar deg, flytter saman med ein du tidlegare har vore gift med eller har barn med, eller får barn med ein sambuar.",
+                    English to "You lose the right to survivor's rights if you marry, move in with a person with whom you were previously married, or give birth to a child with your cohabitee."
```

Comment:
Can't assume that they have a cohabitee.

Reply:
Correct, har oppdaterte teksten.